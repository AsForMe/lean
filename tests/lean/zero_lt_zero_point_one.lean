example : Â¬ 0 < 0.1 := sorry