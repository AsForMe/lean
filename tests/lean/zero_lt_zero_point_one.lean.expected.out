zero_lt_zero_point_one.lean:1:16: error: failed to synthesize type class instance for
⊢ has_div ℕ
